modules/tutorials/pages/gsql-101/get-set.adoc

```diff
@@ -54,7 +64,7 @@ Dan,Kevin,2015-12-30
 
 
 
-=== Prepare Your TigerGraph Environment +++<a id="GSQL101-dropallPrepareYourTigerGraphEnvironment">++++++</a>+++
+=== Prepare your TigerGraph environment +++<a id="GSQL101-dropallPrepareYourTigerGraphEnvironment">++++++</a>+++
```

Comment:
you can delete these passthrough anchor tags - they were from gitbook which was our old platform